src/CONST.js

```diff
@@ -841,6 +841,9 @@ const CONST = {
             // eslint-disable-next-line max-len
             USER_CAMERA_CONSENT_DENIED: 'Unexpected result Intent. It might be a result of incorrect integration, make sure you only pass Onfido intent to handleActivityResult. It might be due to unpredictable crash or error. Please report the problem to android-sdk@onfido.com. Intent: null \n resultCode: 0',
         },
+
+        // This is required as the LOCALES has es-ES and we need to map it to es_ES for Onfido
+        ES_ES_LOCALE: 'es_ES',
```

Comment:
It's better if we place this with the other `LOCALES`.

https://github.com/Expensify/App/blob/b4ed202dc9ae50c7e28e874ff38594cdb6cda2b8/src/CONST.js#L909-L915

and name it `ES_ES_ONFIDO`.